tests/fsharp/typecheck/sigs/neg04.bsl

```diff
@@ -69,55 +69,55 @@ neg04.fs(121,12,121,16): typecheck error FS0193: Type constraint mismatch. The t
     R    
 is not compatible with type
     IBar    
-The type 'R' is not compatible with the type 'IBar'
+
 
 neg04.fs(126,12,126,16): typecheck error FS0193: Type constraint mismatch. The type 
     U    
 is not compatible with type
     IBar    
-The type 'U' is not compatible with the type 'IBar'
```

Comment:
Seems we always gave the error twice. Probably because it is so important to get your types right. 
Anyways, I fixed this as well.